django/db/models/base.py

```diff
@@ -775,6 +784,20 @@ def save(
         that the "save" must be an SQL insert or update (or equivalent for
         non-SQL backends), respectively. Normally, they should not be set.
         """
+
+        # RemovedInDjango60Warning
+        if args:
+            warnings.warn(
+                "Passing positional arguments to save is deprecated",
+                RemovedInDjango60Warning,
+                stacklevel=2,
+            )
+            for arg, attr in zip(
+                args, ["force_insert", "force_update", "using", "update_fields"]
+            ):
+                if arg:
+                    setattr(self, attr, arg)
```

Comment:
Setting attributes on `self` isn't what you want. Notice this test fails:

```py
    @ignore_warnings(category=RemovedInDjango60Warning)
    def test_save_positional_arguments(self):
        a = Article.objects.create(headline="original", pub_date=datetime(2014, 5, 16))
        a.headline = "changed"
        a.save(False, False, None, ["pub_date"])
        a.refresh_from_db()
        self.assertEqual(a.headline, "original")
```
```
======================================================================
FAIL: test_save_positional_arguments (basic.tests.ModelInstanceCreationTests.test_save_positional_arguments)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/jwalls/django/django/test/utils.py", line 446, in inner
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jwalls/django/tests/basic/tests.py", line 205, in test_save_positional_arguments
    self.assertEqual(a.headline, "original")
AssertionError: 'changed' != 'original'
- changed
+ original


----------------------------------------------------------------------
Ran 1 test in 0.007s
```

Reply:
Thanks for notice me this issue. I was in a hurry and I didn't realize that I had to assign parameter of a method and not object attributes. I am going to fix this